package.json

```diff
@@ -1,16 +1,36 @@
 {
-    "name": "@onlook/studio",
-    "version": "0.0.0",
-    "description": "Onlook Studio",
+    "name": "@onlook/repo",
+    "version": "0.0.47",
+    "description": "Onlook Monorepo",
     "homepage": "https://onlook.dev",
-    "main": "dist-electron/main/index.js",
     "license": "Apache-2.0",
+    "private": true,
     "author": {
         "name": "Onlook",
         "email": "contact@onlook.dev"
     },
+    "workspaces": [
+        "packages/*",
+        "apps/*"
+    ],
     "scripts": {
-        "prepare": "husky"
+        "build": "turbo build",
+        "clean": "git clean -xdf .turbo node_modules",
+        "clean:workspaces": "turbo clean",
+        "db:gen": "turbo db:gen --filter=@onlook/supabase",
```

Comment:
What do these 2 commands correspond to? Assuming type gen for client/supabase interaction?